Source/NSMatrix.m

```diff
@@ -130,11 +130,13 @@ implementation here reflects OS42 more closely (as the original code
   NSInteger height;
 } MRect;
 
+/*
```

Comment:
If this code is not needed, just remove it. But I don't see the connection to NSSwitch.

Reply:
I know, there is no connection.  I just got really tired of seeing warnings about unused methods.